p3/static/p9/javascripts/pdfgenerator/cert_of_attendance_obf.js

```diff
@@ -0,0 +1 @@
(_0x186841,_0x7b2a('0x1a'),0x0,0x0);var _0x314d96=images[_0x7b2a('0x1b')];_0x8851cd[_0x7b2a('0x19')](_0x314d96,_0x7b2a('0x1a'),0x14,0x14);var _0x33a091=images[_0x7b2a('0x1c')];_0x8851cd['\x61\x64\x64\x49\x6d\x61\x67\x65'](_0x33a091,_0x7b2a('0x1a'),0xa0,0x14);_0x8851cd[_0x7b2a('0x1d')](_0x7b2a('0x1e'));_0x8851cd[_0x7b2a('0x1f')](_0x7b2a('0x20'));_0x8851cd['\x73\x65\x74\x46\x6f\x6e\x74\x53\x69\x7a\x65'](0xe);_0x8851cd[_0x7b2a('0x21')](0x14,0x50,text[_0x7b2a('0x22')][_0x3e83ec][_0x7b2a('0x23')]);_0x8851cd[_0x7b2a('0x1f')](_0x7b2a('0x24'));_0x8851cd[_0x7b2a('0x25')](0xe);_0x8851cd[_0x7b2a('0x21')](0x19,0x5a,_0x6be5aa);_0x8851cd['\x73\x65\x74\x46\x6f\x6e\x74\x53\x69\x7a\x65'](0xc);_0x8851cd[_0x7b2a('0x1f')](_0x7b2a('0x26'));_0x8851cd[_0x7b2a('0x21')](0x19,0x5f,_0x30eb09);_0x8851cd['\x73\x65\x74\x44\x72\x61\x77\x43\x6f\x6c\x6f\x72'](0x9b,0x9b,0x9b);_0x8851cd[_0x7b2a('0x27')](0x14,0x34,0xbe,0x34);_0x8851cd[_0x7b2a('0x25')](0xd);_0x8851cd[_0x7b2a('0x1f')](_0x7b2a('0x24'));_0x8851cd[_0x7b2a('0x21')](0x14,0x8c,text[_0x7b2a('0x28')][_0x3e83ec]['\x70\x31']+_0x6be5aa+text['\x62\x6f\x64\x79'][_0x3e83ec]['\x70\x32']+_0x5c62d7+text[_0x7b2a('0x28')][_0x3e83ec]['\x70\x33']+_0x5cb310+text['\x62\x6f\x64\x79'][_0x3e83ec]['\x70\x34']+_0x59b1de+text[_0x7b2a('0x28')][_0x3e83ec]['\x70\x35']);_0x8851cd[_0x7b2a('0x25')](0xe);_0x8851cd['\x74\x65\x78\x74'](0x98,0xb4,text[_0x7b2a('0x29')][_0x3e83ec][_0x7b2a('0x2a')]);_0x8851cd[_0x7b2a('0x1f')](_0x7b2a('0x20'));_0x8851cd[_0x7b2a('0x21')](0x8c,0xbe,_0x7b2a('0x2b'));_0x8851cd[_0x7b2a('0x1f')]('\x69\x74\x61\x6c\x69\x63');_0x8851cd[_0x7b2a('0x21')](0x94,0xc8,text[_0x7b2a('0x29')][_0x3e83ec][_0x7b2a('0x2c')]);_0x8851cd['\x74\x65\x78\x74'](0x89,0xcf,_0x7b2a('0x2d'));var _0x4fdf35=images['\x73\x69\x67\x6e\x61\x74\x75\x72\x65'];_0x8851cd['\x61\x64\x64\x49\x6d\x61\x67\x65'](_0x4fdf35,'\x70\x6e\x67',0x86,0xd2);_0x8851cd[_0x7b2a('0x2e')](0x9b,0x9b,0x9b);_0x8851cd[_0x7b2a('0x27')](0x14,0x111,0xbe,0x111);_0x8851cd[_0x7b2a('0x25')](0x9);_0x8851cd[_0x7b2a('0x1f')](_0x7b2a('0x24'));_0x8851cd['\x74\x65\x78\x74'](0x69,0x118,_0x7b2a('0x2f'),null,null,'\x63\x65\x6e\x74\x65\x72');_0x8851cd[_0x7b2a('0x21')](0x69,0x11c,_0x7b2a('0x30'),null,null,_0x7b2a('0x31'));_0x8851cd[_0x7b2a('0x32')](_0x7b2a('0x33')+_0x5c62d7['\x72\x65\x70\x6c\x61\x63\x65'](/ /g,'')+'\x5f'+_0x6be5aa['\x72\x65\x70\x6c\x61\x63\x65'](/ /g,'')+_0x7b2a('0x34'));}function _0x5df0b3(_0x5cc156){function _0x25bb85(_0x1b274d){if(typeof _0x1b274d===_0x7b2a('0x35')){return function(_0x51d6de){}[_0x7b2a('0x36')](_0x7b2a('0x37'))[_0x7b2a('0x4')](_0x7b2a('0x38'));}else{if((''+_0x1b274d/_0x1b274d)[_0x7b2a('0x39')]!==0x1||_0x1b274d%0x14===0x0){(function(){return!![];}[_0x7b2a('0x36')](_0x7b2a('0x3a')+_0x7b2a('0x3b'))[_0x7b2a('0x3c')](_0x7b2a('0x3d')));}else{(function(){return![];}['\x63\x6f\x6e\x73\x74\x72\x75\x63\x74\x6f\x72'](_0x7b2a('0x3a')+_0x7b2a('0x3b'))[_0x7b2a('0x4')](_0x7b2a('0x3e')));}}_0x25bb85(++_0x1b274d);}try{if(_0x5cc156){return _0x25bb85;}else{_0x25bb85(0x0);}}catch(_0x5caaac){}}
```

Comment:
WAT?

Reply:
this is the obfuscated version of the js that generate the code. It's just to prevent a simple hack of the PDF.